cypress/e2e/tests/migration/archetypes/assess_review.test.ts

```diff
@@ -50,13 +50,16 @@ describe(["@tier1"], "Archetype assessment and review tests", () => {
             null,
             stakeholders
         );
+        Archetype.open(true);
         archetype.create();
         cy.wait(2 * SEC);
 
         archetype.perform_assessment("low", stakeholders);
+        Archetype.open(true);
         cy.wait(2 * SEC);
         archetype.validateAssessmentField("Low");
```

Comment:
Pls add the Polarion TC here and in the description.
Polarion MTA-518

Reply:
done